livekit-ffi/protocol/room.proto

```diff
@@ -252,14 +252,15 @@ message RoomEvent {
     ParticipantMetadataChanged participant_metadata_changed = 15;
     ParticipantNameChanged participant_name_changed = 16;
     ConnectionQualityChanged connection_quality_changed = 17;
-    DataReceived data_received = 18;
+    DataReceived data_received = 18 [deprecated=true];
```

Comment:
Feel free to delete instead of deprecating. We can't break the protocol with the FFI

Reply:
Should I just remove the field or set it as `reserved`?

Reply:
remove is OK